tests/metrics/genai/prompts/test_v1.py

```diff
@@ -84,22 +94,38 @@ def test_evaluation_model_output():
         - Score 5: the answer correctly answer the question and not missing any major aspect
 
     Examples:
-        Input: This is an input
-        Provided output: This is an output
-        Provided ground_truth: This is an output
-        Score: 4
-        Justification: This is a justification
-
-        Input: This is an example input 2
-        Provided output: This is an example output 2
-        Provided ground_truth: This is an output
-        Score: 4
-        Justification: This is an example justification 2
-
-    And you'll need to submit your grading for the correctness of the output,
-    using the following in json format:
-    Score: [your score number for the correctness of the output]
-    Justification: [your step by step reasoning about the correctness of the output]
+        Input:
+        This is an input
+
+        Output:
+        This is an output
+
+        Additional information used by the model:
+        key:
+        ground_truth
+        value:
+        This is an output
+
+        score: 4
+        justification: This is a justification
+
+
+        Input:
+        This is an example input 2
+
+        Output:
+        This is an example output 2
+
+        Additional information used by the model:
+        key: ground_truth
+        value: This is an output
+
+        score: 4
+        justification: This is an example justification 2
+
+    You must output a JSON object with the following fields:
+    score: Your numerical score for the model's correctness based on the rubric
+    justification: Your step-by-step reasoning about the model's correctness score
```

Comment:
Can we move this up above the examples so that it's included with the rest of the instructions?

Reply:
@suanmiao mentioned that we need to put this at the bottom so the LLM knows that the expected task at the end. Or else it will read a bunch of examples and do nothing about it. Specially with other non open-ai models, this is very important. That is why I left it at the end

Reply:
are we sure? Putting examples at the end is super common

Reply:
@suanmiao what's the evidence here?

Reply:
> are we sure? Putting examples at the end is super common
@dbczumar can you share some examples for the same? We can compare your examples v/s @suanmiao to see which evidence is better? 
The only information I have read is that LLM care about starting and ending statements more than in between. So making sure that the LLM returns back an output is more important than examples i think. What do you think?